.github/workflows/markdownlint.yml

```diff
@@ -2,7 +2,7 @@ name: markdownlint
 
 on:
   push:
-    branches: [ main, metrics ]
+    branches: [ long-lines-linting ]
```

Comment:
intentional? I guess not..

Reply:
Sure no... I have attached wrong file to the first commit.
It should be in another one, used to execute pipeline on fork.

Fixed afc2d71a4d3091e62cc9a9721edff8e8a3b0c43d

Reply:
It was used to test CI pipeline on my fork. It should be included in the commit which is not part of this PR.

Fixed in afc2d71a4d3091e62cc9a9721edff8e8a3b0c43d